src/PAModel/Microsoft.PowerPlatform.Formulas.Tools.csproj

```diff
@@ -17,33 +17,18 @@
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <Description>Preview Release: This takes a Canvas App (.msapp file) and converts to and from text files that can be checked into source control.</Description>
     <PackageReleaseNotes>
-        Notice: 
+        Notice:
             This package is a preview release. - Use at your own risk.
```

Comment:
nit: "This package is a preview release - use at your own risk."

Removed period after release and changed "Use" to lowercase.